tests/conftest.py

```diff
@@ -3926,7 +3929,7 @@ def factory(
         )
         storageclass = storageclass or pvc_obj.backed_sc
         access_mode = access_mode or pvc_obj.get_pvc_access_mode
-        volume_mode = volume_mode or getattr(pvc_obj, "volume_mode", None)
+        volume_mode = volume_mode or pvc_obj.get_pvc_vol_mode
```

Comment:
any reason for this change?

Reply:
getattr line was not working, it was giving an error, so I used the builtin function of pvc class to get volume mode.